src/Umbraco.Web.UI.Client/src/packages/core/workspace/entity-detail/entity-detail-workspace-base.ts

```diff
@@ -167,6 +167,7 @@ export abstract class UmbEntityDetailWorkspaceContextBase<
 	}
 
 	async load(unique: string) {
+		if (unique === this.getUnique()) return;
```

Comment:
Currently the build breaks because this line result in this method can return "undefined".